workflow/util/path/search.go

```diff
@@ -3,20 +3,18 @@ package path
 import (
 	"fmt"
 	"os"
+	"os/exec"
 	"path/filepath"
-	"strings"
 )
 
 func Search(name string) (string, error) {
 	if filepath.IsAbs(name) {
 		return name, nil
 	}
-	envPath := os.Getenv("PATH")
-	for _, dir := range strings.Split(envPath, ":") {
-		absName := filepath.Join(dir, name)
-		if _, err := os.Stat(absName); err == nil {
-			return absName, nil
-		}
+
+	if absName, err := exec.LookPath(name); err == nil {
```

Comment:
I didn't know about this - we can refactor to remove `path.Search` - please!

Reply:
i.e. inline

Reply:
inlined